spec/controllers/admin/graphs_controller_spec.rb

```diff
@@ -45,19 +53,32 @@
         expect(assigns(:end_at)).to be_within(1.minute).of Time.now
         expect(assigns(:group_period)).to eq "month"
       end
-      context "passed date and time" do
```

Comment:
Looks like this update removed the existing user test. 

The user test should still exist!

Also, it would be great if you tested something more useful than just `keys.count > 0` - I know I didn’t initially, but 🤗

Reply:
was this resolved?

Reply:
Yeah, I still have the user and payment test on the branch I just pushed up.